### ISSUE TITLE
Add an option to cancel current deployment 

### ISSUE THREAD

**Issue by gmontalvoy** [openshift/machine-config-operator#1248]
Hi there,

Adding a new config on a MachineConfigPool will try to apply this to the entire pool.
Imagine you added a validated but wrong file config that cause workers to fails in any way, at this moment you can pause the deployment but there is no way to revert it, even if you reload the old and working configuration, the MC will apply the failed config to the entire pool spreading the wrong config before start to apply the second one.

TL DR: any way to cancel the current deployment/sync?

Thanks

**Comment by runcom** [comment]
> TL DR: any way to cancel the current deployment/sync?

delete the MC? if the rollout isn't finished, it'll stop since you no longer want that.



> Imagine you added a validated but wrong file config that cause workers to fails in any way

if the worker fails at a system/kube level, the MCO won't roll out to other nodes, this is how it works.

**Comment by gmontalvoy** [comment]
Thank you Antonio,

> delete the MC? if the rollout isn't finished, it'll stop since you no longer want that.
> if the worker fails at a system/kube level, the MCO won't roll out to other nodes, this is how it works.

Didn't know about that validation at the first worker, thank you.

*closing*
